scico/functional/_norm.py

```diff
@@ -178,13 +177,17 @@ def prox(
             \end{cases}
 
         Args:
-            x:  Input array :math:`\mb{x}`.
+            v:  Input array :math:`\mb{v}`.
             lam: Proximal parameter :math:`\lambda`.
+            kwargs: Additional arguments that may be used by derived
+                classes. These include ``x0``, an initial guess for the
+                minimizer.
```

Comment:
Perhaps worth providing a bit more details on "the minimizer"?

Reply:
Good point, `x0` shouldn't be mentioned at all in many of these cases. In fact, `kwargs` is not often used, but I'm not sure how better to document that.